src/wp-admin/includes/image.php

```diff
@@ -414,6 +519,13 @@ function _wp_make_subsizes( $new_sizes, $file, $image_meta, $attachment_id ) {
 		return $image_meta;
 	}
 
+	// Calculate the primary and additional mime types to generate.
+	$upload_mime_transforms = wp_upload_image_mime_transforms( $attachment_id );
+	$mime_type              = wp_get_image_mime( $file );
+	$mimes_to_generate      = _wp_get_primary_and_additional_mime_types( $upload_mime_transforms, $attachment_id, $mime_type );
+	$primary_mime_type      = isset( $mimes_to_generate['primary_mime_type'] ) ? $mimes_to_generate['primary_mime_type'] : array();
+	$additional_mime_types  = isset( $mimes_to_generate['additional_mime_types'] ) ? $mimes_to_generate['additional_mime_types'] : array();
```

Comment:
See above, this can be simplified similarly to a single line.

Reply:
👍🏼  will update & adjust primary default

Reply:
updated